target/linux/qualcommax/ipq60xx/base-files/etc/board.d/02_network

```diff
@@ -23,9 +27,29 @@ ipq60xx_setup_interfaces()
 	esac
 }
 
+ipq60xx_setup_macs()
+{
+	local board="$1"
+	local lan_mac=""
+	local wan_mac=""
+	local label_mac=""
+
+	case $board in
+	linksys,mr7350)
+		wan_mac=$(macaddr_add "$(mtd_get_mac_ascii devinfo hw_mac_addr)")
+		lan_mac=$(macaddr_add "$(mtd_get_mac_ascii devinfo hw_mac_addr)" 1)
```

Comment:
Are you sure this address is correct?

Reply:
Yeah, `mtd_get_mac_ascii devinfo hw_mac_addr` provides the same address as on the router label.

Reply:
I remember the lan/wan mac addresses are the same.
```
	linksys,mr7350)
		label_mac=$(mtd_get_mac_ascii devinfo hw_mac_addr)
		lan_mac=$label_mac
		wan_mac=$label_mac
		;;
```

Reply:
Yeah, I was unsure about this. Thanks.